pkg/ddl/tests/partition/multi_domain_test.go

```diff
@@ -233,8 +233,8 @@ func TestMultiSchemaDropListColumnsDefaultPartition(t *testing.T) {
 			// tkO see non-readable/non-writable p0 partition, and should try to read from p1
 			// in case there is something written to overlapping p1
 			tkO.MustContainErrMsg(`insert into t values (1,1,1)`, "[table:1526]Table has no partition for value matching a partition being dropped, 'p0'")
-			tkNO.MustContainErrMsg(`insert into t values (1,1,1)`, "[kv:1062]Duplicate entry '1' for key 't.a_2'")
-			tkO.MustContainErrMsg(`insert into t values (101,101,101)`, "[kv:1062]Duplicate entry '101' for key 't.a_2'")
+			tkNO.MustContainErrMsg(`insert into t values (1,1,1)`, "[kv:1062]Duplicate entry '1' for key")
+			tkO.MustContainErrMsg(`insert into t values (101,101,101)`, "[kv:1062]Duplicate entry '101")
```

Comment:
why change this part?

Reply:
The tests were not stable, so since I stumbled upon the test failures, I made the tests more stable, by allowing errors on any existing unique key.

Reply:
seems you are the author of those cases, it's ok if the key name is not the part of the thing we want to test

Reply:
This is OK.

The test is to verify that unique keys are not violated (or rather that transactions for both current schema version and previous schema version sees the same data), and there are other places where this type of test results checks only have the value and not a specific unique key.